src/components/menu/components/dishes/Dishes.js

```diff
@@ -0,0 +1,63 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+
+//Components
+import CategoryDishes from './components/categoryDishes';
+
+//CSS
+import './dishes.css';
+
+function renderCategoryDishes(category, dishes, quantities, handleEvents) {
+  return (
+    <CategoryDishes
+      key={category}
+      category={category}
+      dishes={dishes[category]}
+      quantities={quantities}
+      handleEvents={handleEvents}
+    />
+  );
+}
+
+function renderCategoriesDishes(dishes, quantities, handleEvents) {
+  const categories = Object.keys(dishes);
+  return categories.map((category) =>
+    renderCategoryDishes(category, dishes, quantities, handleEvents)
+  );
+}
+function Dishes(props) {
+  const { dishes, quantities, handleEvents } = props;
+  return (
+    <div className='dishes'>
+      {renderCategoriesDishes(dishes, quantities, handleEvents)}
```

Comment:
you have. two functions for rendering categoriesDish instead you can define the `renderCategoryDishes` inside `dishes` function and use _map of `lodash` which will give the key and object to the called function automatically